src/+otp/+kuramotosivashinsky/+presets/Canonical.m

```diff
@@ -0,0 +1,29 @@
+classdef Canonical < otp.kuramotosivashinsky.KuramotoSivashinskyProblem
+    
+    methods
+        function obj = Canonical(varargin)
+            
+            p = inputParser;
+            addParameter(p, 'Size', 64, @isscalar);
+            addParameter(p, 'L', 25, @isscalar);
```

Comment:
I would remove the validation function and leave it to `validateNewState`

Reply:
Also, having the size `n` in the parameters is redundant.  There can be inconsistencies with the size of `Y0` and `n`.  I think we should prefer `Problem.NumVars`